app/models/user_role_permissions.rb

```diff
@@ -1,27 +1,45 @@
 class UserRolePermissions
-  attr_reader :component_management, :team_management, :user_management,
-              :read_components, :update_profile, :change_password
+  attr_reader :component_management, :team_management, :user_management, :event_managemnt,
```

Comment:
there's a typo in `:event_managemnt`, but do we need to be that granular? ... events can be viewed, nothing else, and viewed by GDS users only...

Reply:
Typo is throughout project FYI